lesson1/src/stacks_queues/stackProblems.js

```diff
@@ -19,12 +19,18 @@ class StackProblems {
    */
   static balancedParenthesis(string) {
     let stack = new Stack();
```

Comment:
not using a stack on stackProblems.js sounds like a bad idea... 😝, try thinking how to solve it with the stack (it will be more efficient) and you'll learn how calculators work ➕